vibrato/src/dictionary/character.rs

```diff
@@ -1,9 +1,10 @@
-mod builder;
-
+use std::collections::BTreeMap;
 use std::fmt;
+use std::io::{prelude::*, BufReader, Read};
```

Comment:
It is better not to use `prelude::*` because it makes difficult to know what is imported.

Reply:
Fixed https://github.com/daac-tools/vibrato/pull/77/commits/1b49608a820e40c866ca0e89b1dac4d05881b2d5